app/scripts/controllers/exportImport.js

```diff
@@ -32,7 +32,7 @@ angular.module('openhimConsoleApp')
       $scope.showRecordOptions.ContactGroups = false;
       $scope.showRecordOptions.Keystore = false;
     };
-    
```

Comment:
The only problem with autoformatting is that it makes the relevant changes harder to review since there are unrelated changes in the diff. If you are going to make formatting changes then they should be done in a separate commit so that they can be reviewed independently of the actual change.